src/sumOfOther.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function sumOfOther(arr) { 
+	let sum = arr.reduce(function(a, b) {
```

Comment:
And here are the consequences.
In this case the _eslint_ would highlight the error here saying to use `const` instead of `let`, because don't re-assign this variable.